test/k8s-integration/main.go

```diff
@@ -157,26 +159,27 @@ func handle() error {
 
 	// If running in Prow, then acquire and set up a project through Boskos
 	if *inProw {
-		project, _ := testutils.SetupProwConfig(*boskosResourceType)
-
 		oldProject, err := exec.Command("gcloud", "config", "get-value", "project").CombinedOutput()
+		*stagingImage = fmt.Sprintf("gcr.io/%s/gcp-persistent-disk-csi-driver", strings.TrimSpace(string(oldProject)))
```

Comment:
staging image should only be overridden if `doDriverBuild` = true. Can we actually pull L173 out of the linux block so we set it in one place for both windows and linux?

Reply:
updated